docs/2.0/docs/patcher/tutorials/applying-first-patch.md

```diff
@@ -1,3 +1,161 @@
+</details>
+
+From here hit `enter` and patcher will apply the update:
+
+![Patcher update shows that it found a patch to be applied and what version it is incrementing to](/img/patcher/tutorials/patcher_update_in_progress.png)
+
+![Patcher Update completion notice](/img/patcher/tutorials/patcher_update_complete.png)
+
+You can now inspect the filesystem and see the results of the patch.  In this case it resulted in changing the version of `patcher-test` from `v0.10.3` to `v0.10.5` and added a new required argument `sampleinput` to the unit. This change can now be committed and pushed to your repository.
+
+![Patcher Update completion notice](/img/patcher/tutorials/patcher_update_results.png)
+
+## Applying a patch using Patcher via GitHub Actions
+
+Choose a directory named: <CustomizableValue id="DIRECTORY" /> in your repository where we'll add a new infrastructure unit.  In that folder create a file called `terragrunt.hcl` with the following content:
+
+```hcl title="$$DIRECTORY$$/terragrunt.hcl"
+
+terraform {
+  source = "git@github.com:gruntwork-io/terraform-aws-utilities.git//modules/patcher-test?ref=v0.10.3"
+}
+
+# Include the root `terragrunt.hcl` configuration, which has settings common across all environments & components.
+include "root" {
+  path = find_in_parent_folders()
+}
+```
+
+Now we'll push this up to your repository.
+
+```bash
+$ git checkout -b patcher-test-update
+$ git add $$DIRECTORY$$/terragrunt.hcl
+$ git commit -m "Adding a new infrastructure unit that uses the patcher-test module"
+$ git push -u origin patcher-test-update
+
+```
+
+Now navigate to your repository in GitHub and create a new pull request for the `patcher-test-update` branch.  In order for Patcher to identify this unit as needing an update, it first has to be merged to main, so go ahead and merge the pull request.
+
+:::info
+In a real world use case, you are unlikely to create a new unit that immediately is out of date, and instead Patcher would on a schedule and detect updates to modules some time after they have been merged.  For the purpose of this tutorial, however, we're intentionally merging an out of date module to main so we can trigger Patcher to update it immediately.
+:::
+
+Once the code has been merged to main, we can now directly trigger Patcher to scan for updateable modules.
+
+<details>
+<summary>If you don't already have Patcher installed in your repository, you can add the following GitHub Actions workflow:</summary>
+
+```yaml title=".github/workflows/patcher.yml"
+
+name: Patcher - Update Dependencies
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - main
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: gruntwork-io/patcher-action@v2
+        with:
+          github_token: ${{ secrets.PATCHER_GRUNTWORK_READ_TOKEN }}
```

Comment:
There isn't a mention of setting up secrets prior to this. The linked installation doc in the pre-req also do not cover this.

Reply:
Yeah, we have a gap right now - no docs whatsoever on installing patcher as GH action.  It's implied that you get it for automatically w/DF.  I'll file a ticket and update the copy here.